packages/sdk/src/methods/token.ts

```diff
@@ -0,0 +1,57 @@
+import { BaseAssetId, bufferFromString, sha256 } from 'fuels';
+import { config } from '../config';
+import { getRegistryContract } from '../setup';
+import {
+  type ProviderParams,
+  assertValidDomain,
+  getProviderFromParams,
+} from '../utils';
+
+type TokenInfo = {
+  name: string;
+  image: string;
+  symbol: string;
+  subId: string;
+  contractId: string;
+};
+
+/**
+ * Retrieves token data.
+ *
+ * @param {string} handle - The handle of the token.
+ * @param {ProviderParams} params - The provider parameters.
+ *
+ * @return {Promise<TokenInfo|undefined>} - The token information.
+ */
+export async function tokenInfo(handle: string, params: ProviderParams) {
+  const handleName = assertValidDomain(handle);
+  const provider = await getProviderFromParams(params);
+
+  const { registry } = await getRegistryContract({
+    storageId: config.STORAGE_CONTRACT_ID!,
+    provider,
+  });
+
+  const { value: tokenImage } = await registry.functions
+    .image_url(handleName)
+    .get();
+
+  if (!tokenImage) {
+    return undefined;
+  }
+
+  const { value: tokenName } = await registry.functions
+    .name({ value: BaseAssetId })
+    .get();
+  const { value: tokenSymbol } = await registry.functions
+    .symbol({ value: BaseAssetId })
+    .get();
+
+  return {
+    name: tokenName,
+    image: tokenImage,
+    symbol: tokenSymbol,
+    subId: sha256(bufferFromString(handleName)),
```

Comment:
Error on try convert the `handleName` with value `mydomain22` to ASCII bytes.

With value `mydomain` works.

Reply:
```bash
Uncaught (in promise) DOMException: Failed to execute 'atob' on 'Window': The string to be decoded is not correctly encoded.
    at bufferFromString
```